docs/src/selenium-grid.md

```diff
@@ -129,10 +129,10 @@ Here is an example of running selenium standalone and connecting Playwright to i
 First start Selenium.
 
 ```bash
-docker run -d -p 4444:4444 --shm-size="2g" -e SE_NODE_GRID_URL="http://localhost:4444" selenium/standalone-chrome:4.3.0-20220726
+docker run -d -p 4444:4444 --shm-size="2g" -e SE_NODE_GRID_URL="http://localhost:4444" selenium/standalone-chrome:latest
 
 # Alternatively for arm architecture
-docker run -d -p 4444:4444 --shm-size="2g" -e SE_NODE_GRID_URL="http://localhost:4444" seleniarm/standalone-chromium:103.0
+docker run -d -p 4444:4444 --shm-size="2g" -e SE_NODE_GRID_URL="http://localhost:4444" selenium/standalone-chromium:latest
```

Comment:
Let's use `standalone-chromium:latest` for all arcs and just remove these two lines if arm is supported by default.

Reply:
Ok, I updated it to a single line with chromium.